Dockerfile

```diff
@@ -346,34 +355,42 @@ ARG PRODUCT
 ENV PRODUCT=${PRODUCT}
 ARG DEFAULT_PRODUCT_LICENSE
 ENV DEFAULT_PRODUCT_LICENSE=${DEFAULT_PRODUCT_LICENSE}
+ARG COMPANY_NAME
+ENV COMPANY_NAME=${COMPANY_NAME}
```

Comment:
Was thinking again about this one, and (also per our chats on slack), perhaps `PACKAGER_NAME` is a more descriptive name for this (also see https://github.com/docker/cli/pull/3500)